assets/queries/cloudFormation/cloudfront_minimum_protocol_tls1.2/query.rego

```diff
@@ -0,0 +1,37 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].Resources[name]
+  resource.Type == "AWS::CloudFront::Distribution"
+  properties := resource.Properties
+  object.get(properties.DistributionConfig, "ViewerCertificate", "undefined") == "undefined"
+
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("Resources.%s.Properties.DistributionConfig",[name]),
+                "issueType":		"MissingAttribute",  
+                "keyExpectedValue": sprintf("Resources.%s.Properties.DistributionConfig.ViewerCertificate' is defined",[name]),
+                "keyActualValue": 	sprintf("Resources.%s.Properties.DistributionConfig.ViewerCertificate' is undefined",[name]),
+              }
+}
+
+CxPolicy [ result ] {
+  resource := input.document[i].Resources[name]
+  resource.Type == "AWS::CloudFront::Distribution"
+  properties := resource.Properties
+  protocolVer := properties.DistributionConfig.ViewerCertificate.MinimumProtocolVersion
+  not contains(["TLSv1.2_2018", "TLSv1.2_2019"], protocolVer) 
```

Comment:
I advise you to change the function name to `containsProtocolVersion` in order to avoid confusion with the built-in function `contains`